src/test/java/lambda/part1/exercise/Exercise3.java

```diff
@@ -0,0 +1,71 @@
+package lambda.part1.exercise;
+
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.arrayContaining;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+class Exercise3 {
+
+    @Test
+    void sortPersonsByAgeUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        // TODO use Arrays.sort + expression-lambda
+        Arrays.sort(persons, (p1, p2) -> Integer.compare(p1.getAge(), p2.getAge()));
+
+        assertThat(persons, is(arrayContaining(
+                new Person("Иван", "Мельников", 20),
+                new Person("Николай", "Зимов", 30),
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45)
+        )));
+    }
+
+    @Test
+    void sortPersonsByLastNameThenFirstNameUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        // TODO use Arrays.sort + statement-lambda
+        Comparator<Person> personByFirstNameComparator = (p1, p2) -> p1.getFirstName().compareToIgnoreCase(p2.getFirstName());
```

Comment:
You need to use here Arrays.sort + statement-lambda, have a look at the TODO inside the method.

Reply:
Ok, I got it, Just fixed.

Reply:
I meant something like that:
```
Arrays.sort(persons, (p1, p2) -> {
    int i = p1.getLastName().compareToIgnoreCase(p2.getLastName());
    return i == 0 ? p1.getFirstName().compareToIgnoreCase(p2.getFirstName()) : i;
});
```
But nvmd, I'll accept your solution.